sktime/forecasting/base/_base.py

```diff
@@ -1687,6 +1688,7 @@ def _check_fh(self, fh):
         Parameters
         ----------
         fh : None, int, list, np.ndarray or ForecastingHorizon
+        non_vectorized_check : bool. Check the cutoff of fh if True.
```

Comment:
wrong param name - is the content description also correct?

Reply:
for sure, just removed this as per your suggestion below